<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert_complex</class>
 <widget class="QDialog" name="insert_complex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 65, 66);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>430</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 92, 93);</string>
    </property>
    <property name="title">
     <string>Описание упражнения</string>
    </property>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>450</y>
       <width>226</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #4CAF50;          /* Основной зеленый */
    border: 2px solid #388E3C;         /* Темно-зеленая граница */
    border-radius: 8px;                /* Закругление углов */
    color: white;
    font-size: 24px;                   /* Увеличиваем размер &quot;+&quot; */
    font-weight: bold;
    min-width: 40px;
    min-height: 40px;
    padding: 0px;
    margin: 2px;
}

QPushButton:hover {
    background-color: #66BB6A;          /* Светло-зеленый при наведении */
    border-color: #4CAF50;
}

QPushButton:pressed {
    background-color: #2E7D32;         /* Темно-зеленый при нажатии */
    border-color: #1B5E20;
}</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>311</width>
       <height>403</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);
border-radius: 8px;</string>
          </property>
          <property name="text">
           <string>Название упражнения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_title">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);
border-radius: 8px;</string>
          </property>
          <property name="text">
           <string>Группа мышц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_group_muscle">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);
border-radius: 8px;</string>
          </property>
          <property name="text">
           <string>Рабочий вес</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_work_weight">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);
border-radius: 8px;</string>
          </property>
          <property name="text">
           <string>Кол-во подходов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_numb_try">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);
border-radius: 8px;</string>
          </property>
          <property name="text">
           <string>Кол-во повторов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_numb_repit">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
